includes/class-activitypub.php

```diff
@@ -498,6 +499,25 @@ public static function theme_compat() {
 		}
 	}
 
+	/**
+	 * Pre-get option filter for the Authorized Fetch.
+	 *
+	 * @param string $pre The pre-get option value.
+	 *
+	 * @return string If the constant is defined, return the value, otherwise return the pre-get option value.
+	 */
+	public static function pre_option_activitypub_authorized_fetch( $pre ) {
```

Comment:
Maybe this could be moved underneath `pre_option_activitypub_actor_mode` to group them and keep with the order the callbacks are registered?